registry/remote/example_test.go

```diff
@@ -383,7 +383,7 @@ func ExampleRepository_Fetch_manifestByTag() {
 		panic(err)
 	}
 	defer rc.Close() // don't forget to close
-	pulledBlob, err := io.ReadAll(rc)
+	pulledBlob, err := content.ReadAll(rc, descriptor)
```

Comment:
Now that we have `content.ReadAll()`, we don't need to manually verify the fetch content anymore. (See L390-L393 and other similar ones).